package.json

```diff
@@ -31,6 +31,20 @@
   },
   "preferences": [
     {
+      "name": "PROTOCOL",
```

Comment:
Ah, I should have been more specific with my comments earlier.

It might be easier to just have a BASE_URL pref, with the value `http://ideatown.dev:8000`. A little redundancy between that pref and the HOSTNAME pref is totally fine.

I'll just file this as a minor followup bug, so this can land today.